tests/dialects/test_tsql.py

```diff
@@ -2090,3 +2090,19 @@ def test_next_value_for(self):
                 "oracle": "SELECT NEXT VALUE FOR db.schema.sequence_name",
             },
         )
+
+    def test_datetrunc(self):
```

Comment:
None of these tests check the roundtrip; We must include a `"tsql"` entry in either read/write dicts